PLAT/PLAT/Domain/UseCase/AuthUseCase.swift

```diff
@@ -57,6 +57,15 @@ extension AuthUseCase {
         case .failure(let error): return .failure(error)
         }
     }
+    
+    /// 내가 게시한 트랙인지 확인하기
+    func checkMyTrack(currentTrack: Track) -> Bool {
+        if state.user?.id == currentTrack.user.id {
+            return true
+        } else {
+            return false
+        }
+    }
```

Comment:
요렇게 코드 수정할 수도 있을 것 같아요!
```suggestion
    func checkMyTrack(currentTrack: Track) -> Bool {
        return state.user?.id == currentTrack.user.id
    }
```